newIDE/app/src/UI/AlertMessage.js

```diff
@@ -88,7 +88,7 @@ const AlertMessage = ({
                     </React.Fragment>
                   ) : (
                     <React.Fragment>
-                      {kind === 'info' && <Info style={styles.icon} />}
+                      {kind === 'info' && <SquaredInfo style={styles.icon} />}
```

Comment:
you're changing this logo everywhere on purpose?

Reply:
I renamed it because I added a circled one.